ledger/controllers/publisher.js

```diff
@@ -640,16 +640,14 @@ module.exports.initialize = async (debug, runtime) => {
       unique: [ { rulesetId: 1 } ],
       others: [ { type: 1 }, { version: 1 }, { timestamp: 1 } ]
     },
-/* deprecated
```

Comment:
maybe it would be good to add some comments explaining the difference between the various collections here. it's hard to understand the intent / what's still in use without digging through code as is.

a general style comment - I think we should generally avoid commenting out blocks of code like this (and above) in the future. it's easier to grep the code to tell what's still in use and if we ever need to restore them we can simply pull them out of the git history. for cases where it makes more sense to leave the code in place but commented we should leave a descriptive comment. `deprecated` isn't enough, I would have guessed that this block was gone and never coming back and that we were just too lazy to remove it based on that comment. but here we are restoring it - not obvious.

Reply:
agreed.